src/worldObjects.js

```diff
@@ -48,7 +48,7 @@ createNameSpace("realityEditor.worldObjects");
         // when an object is detected, check if we need to add a world object for its server
         realityEditor.network.addObjectDiscoveredCallback(function(object, objectKey) {
             
-            if (object.isWorldObject) {
+            if (object.isWorldObject || object.type === 'world') {
```

Comment:
Why do we need to check for both of these things? Are we migrating away from using .isWorldObject?

Reply:
Yeah there used to not be an object `type` property so we added `isWorldObject`, `isAnchor`, etc.. I want to eventually just use `type` for everything as a more generic property (used for avatar, human, world, anchor, object, and whatever else we come out with in the future). There's probably a more intentional way to phase out isWorldObject, but for now I've just been adding the extra conditional whenever I encounter it